witness/src/Dscp/Witness/Web/Logic.hs

```diff
@@ -241,10 +241,10 @@ checkFairCV =
         }
 
 checkFairCVPDF :: forall ctx m. WitnessWorkMode ctx m => ByteString -> m FairCVAndCheckResult
-checkFairCVPDF pdf = do
+checkFairCVPDF (LBS.fromStrict -> pdf) = do
```

Comment:
Frankly, why the endpoint accepts strict `ByteString` in the first place? :thinking:

Reply:
Made endpoint accept `LByteString`